crates/language-tests/tests/language/functions/array/shuffle.surql

```diff
@@ -14,11 +14,10 @@ value = "[4]"
 value = "[1, 1, 1]"
 
 [[test.results]]
-match = """
-	1 in $result && 2 in $result && 3 in $result && 4 in $result && "text" in $result
-"""
-
+match = "$results CONSTAINSALL [1,2,'text',3,4]"
```

Comment:
```suggestion
match = "$results CONTAINSALL [1,2,'text',3,4]"
```

Reply:
Huh, how that that test run successfully with an invalid operator? That might be a bug in the test suite.